includes/AMP/Canonical_Sanitizer.php

```diff
@@ -2,10 +2,10 @@
 /**
  * Class Canonical_Sanitizer.
  *
- * @package   Google\Web_Stories
```

Comment:
@swissspidy Do we want package removed here?

Reply:
Yes, I deliberately made that change in 6ce0f3a6c1b5785709229f8669ba7157c26e012c

It's really redundant as we have namespaces to indicate "packages".